src/main/java/fi/otavanopisto/kuntaapi/server/integrations/casem/CaseMCacheUpdater.java

```diff
@@ -300,6 +312,34 @@ private String getName(List<ExtendedProperty> extendedProperties) {
     return null;
   }
   
+  private List<FileId> getAttachmentFileIds(OrganizationId organizationId, List<ExtendedProperty> extendedProperties) {
+    List<FileId> result = new ArrayList<>();
+    
+    for (ExtendedProperty extendedProperty : extendedProperties) {
+      String name = extendedProperty.getName();
+      
+      if (EXTENDED_ATTACHMENTS.equals(name) || EXTENDED_AGENDA_ATTACHMENT.equals(name)) {
+        result.addAll(parseAttachmentFileIds(organizationId, extendedProperty.getText()));
+      }
+    }
+    
+    return result;
+  }
+  
+  private List<FileId> parseAttachmentFileIds(OrganizationId organizationId, String text) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "parseAttachmentFileIds" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)